clusterloader2/testing/density/config.yaml

```diff
@@ -61,7 +61,8 @@ steps:
     Params:
       action: gather
 - name: Creating saturation pods
-# Create latency pods
+# TODO: Change this explicite iterations when pod startup problem is fixed!
```

Comment:
Ohh man - how ugly that is...

Can you at least open an issue to trigger reverting that?

Reply:
I meant "to not forget about reverting" :-)

Reply:
Issue #310